### PR TITLE
[WWST-6480] Second batch of DJN changes

### PR DESCRIPTION
**Pull request by PKacprowiczS**
@tpmanley @greens @dkirker @KKlimczukS @MGoralczykS @ZWozniakS @MWierzbinskaS @BJanuszS

### CODE REVIEW

File: devicetypes/smartthings/arrival-sensor.src/arrival-sensor.groovy [hunk 1]
```diff
@@ -23,9 +23,9 @@ metadata {
 		capability "Battery"
 		capability "Health Check"
 
-		fingerprint profileId: "FC01", deviceId: "019A"
-		fingerprint profileId: "FC01", deviceId: "0131", inClusters: "0000,0003", outClusters: "0003"
-		fingerprint profileId: "FC01", deviceId: "0131", inClusters: "0000", outClusters: "0006"
+		fingerprint profileId: "FC01", deviceId: "019A", deviceJoinName: "Presence Sensor"
```
Comment (tpmanley) [inline devicetypes/smartthings/arrival-sensor.src/arrival-sensor.groovy]: These are all SmartThings devices

Comment (PKacprowiczS) [reply devicetypes/smartthings/arrival-sensor.src/arrival-sensor.groovy]: I thought so, but fingerprints have looked to be generic, so I've decided to left it out that way. Anyway, tbd.

Comment (tpmanley) [reply devicetypes/smartthings/arrival-sensor.src/arrival-sensor.groovy]: True, but these fingerprints have the manufacturer specific profile ID so it's highly unlikely that another device would use this DTH besides our first party stuff.

**Review by greens — APPROVED** [review]

File: devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/zigbee-white-color-temperature-bulb.groovy [hunk 2]
```diff
@@ -32,80 +32,80 @@ metadata {
 
 
 		// Generic
-		fingerprint profileId: "0104", deviceId: "010C", inClusters: "0006, 0008, 0300", deviceJoinName: "Generic Color Temperature Light"
+		fingerprint profileId: "0104", deviceId: "010C", inClusters: "0006, 0008, 0300", deviceJoinName: "Light" //Generic Color Temperature Light
 
 		// ABL
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "Samsung Electronics", model: "ABL-LIGHT-Z-001", deviceJoinName: "Ultra Thin Wafer", mnmn: "Samsung Electronics", vid: "ABL-LIGHT-Z-001"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "Samsung Electronics", model: "ABL-LIGHT-Z-001", deviceJoinName: "Samsung Light", mnmn: "Samsung Electronics", vid: "ABL-LIGHT-Z-001" //Ultra Thin Wafer
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/zigbee-white-color-temperature-bulb.groovy:38]: This one in particular should get a second look. I'm fine with merging this as is, but @PKacprowiczS  can you double check this change is correct?

**Review by tpmanley — APPROVED** [review]

File: devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/zigbee-white-color-temperature-bulb.groovy (hunk 2 above)
Comment (JasonJeong33) [reply devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/zigbee-white-color-temperature-bulb.groovy:38]: @tpmanley @PKacprowiczS This device is sold under the Juno brand.
deviceJoinName: "**Juno** Light"

**Pull request merged**